src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -56,19 +83,29 @@ const SubjectsStep = ({ btnsBox }) => {
           />
           <AsyncAutocomplete
             labelField='name'
-            onChange={(_e, newValue) => setSubject(newValue)}
+            onChange={(_e, newValue) => handleSubjectChange(newValue)}
             service={currentSubjectService}
             textFieldProps={subjectTextFieldProps}
             value={subject ? subject._id : null}
             valueField='_id'
           />
           <AppButton
             disabled={!subject}
+            onClick={handleAddSubject}
             sx={styles.addSubjectBtn}
             variant={'tonal'}
           >
             {t('becomeTutor.categories.btnText')}
           </AppButton>
+          <FormHelperText error={subjectsError !== null}>
+            {subjectsError || null}

```

Comment:
`subjectsError` can be `null` or string, so we can write here 
```suggestion
            {subjectsError}

```